piety-promoted-v1/modernized/01-to-the-reader.adoc

```diff
@@ -176,33 +176,40 @@ having already "`their inheritance sealed unto them by the Spirit of God.`"
 
 Some said as follows:
 
-Richard Hubberthorn: '`I know the ground of my salvation,
+[.discourse-part]
+_Richard Hubberthorn:_ '`I know the ground of my salvation,
 and am satisfied forever in my peace with the Lord God.`'
 
-Thomas Loe: '`I am near leaving you, I think; but as well in my spirit as I can desire.
+[.discourse-part]
+_Thomas Loe:_ '`I am near leaving you, I think; but as well in my spirit as I can desire.
 I bless the Lord,
 I never saw more of the glory of God than I have done this day;`' and he sang, '`Glory,
 glory to the Lord.`'
 
-Richard Farnsworth: '`I am filled with the love of God more than I can express;
+[.discourse-part]
+_Richard Farnsworth:_ '`I am filled with the love of God more than I can express;
 and God has really appeared for us.
 If God himself had come down and spoken as a man,
 he could not have spoken more clearly to us than he has done,
 by many testimonies from heaven in his people.`'
 
-Josiah Coale: '`For my part, I have walked in faithfulness with the Lord,
+[.discourse-part]
+_Josiah Coale:_ '`For my part, I have walked in faithfulness with the Lord,
 and I have thus far finished my testimony, and have peace with the Lord,
 and his majesty is with me, and his crown of life is upon me.`'
 
-Francis Howgill: '`As for me, I am well content to die;
+[.discourse-part]
+_Francis Howgill:_ '`As for me, I am well content to die;
 and if any enquire after my latter end,
 let them know that I die in the faith in which I lived and suffered for.`'
 
-William Bayley: '`Tell my friends, I go to my Father and to their Father,
+[.discourse-part]
+_William Bayley:_ '`Tell my friends, I go to my Father and to their Father,
 to my God and to their God; and let not my wife mourn in sorrow,
 for it is well with me.`'
 
-Sarah Beck: '`I feel sweet peace and great joy.
+[.discourse-part]
+_Sarah Beck:_ '`I feel sweet peace and great joy.
 O the joy that is laid up for the righteous:
 who would not fear the Lord! who would not be faithful!`'
 
```

Comment:
I would put a [.small-break] between this last line of Sarah Beck, and the next paragraph, so that the next paragraph doesn't seem like it is still part of what Sarah Beck said.  (I don't expect you to know a lot of these things, but I'm kinda showing you how I think about it, so you can get more of a feel for it)

Reply:
No problem, I appreciate that.